README.md

```diff
@@ -104,7 +104,7 @@ cd mathmagicians-calculator
 npm i && npm start
 ```
 
-Then open [http://localhost:3000/](http://localhost:3000/) to see the app.
+Then open [https://math-magicians-calculator.herokuapp.com/](https://math-magicians-calculator.herokuapp.com/) to see the app.
```

Comment:
I think as the instructions are about development, what a dev would need is to open `localhost:3000`

I suggest substituting for this string `http://localhost:3000/`

Reply:
You suggest that it should be like this 

[http://localhost:3000/](https://math-magicians-calculator.herokuapp.com/)